tensorflow/python/keras/activations.py

```diff
@@ -179,21 +179,28 @@ def softsign(x):
 def relu(x, alpha=0., max_value=None, threshold=0):
   """Rectified Linear Unit.
 
-  With default values, it returns element-wise `max(x, 0)`.
-
-  Otherwise, it follows:
+  The Rectified Linear Unit function is:
```

Comment:
This file should not be in this PR. See the `git checkout` command above to revert.